jenkins/Jenkinsfile-blossom.premerge

```diff
@@ -345,11 +345,12 @@ void checkoutCode(String url, String sha) {
         poll: true,
         scm: [
             $class           : 'GitSCM', branches: [[name: sha]],
-            submoduleCfg     : [],
```

Comment:
Remove this unused option

[For now no use case was found for this option   but to make sure all config options are supported this option was added](https://github.com/wcm-io-devops/jenkins-pipeline-library/blob/master/vars/checkoutScm.md#submodulecfg--optional)